app/lib/guider_importer.rb

```diff
@@ -0,0 +1,75 @@
+class GuiderImporter
+  def initialize(name)
+    @name = name
+  end
+
+  def call
+    User.find_or_create_by!(mapped_attributes)
+  end
+
+  private
+
+  def mapped_attributes
+    { name: name }.tap do |attributes|
+      if mappings.key?(name)
+        attributes[:uid] = mappings[name]
+      else
+        attributes[:disabled] = true
```

Comment:
Probably best we refer to these as `hidden`. Don't know if that needs to change here but just thought I'd point out.